packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -94,7 +94,20 @@ export class DynamoDBModelTransformer extends Transformer {
      */
     public object = (def: ObjectTypeDefinitionNode, directive: DirectiveNode, ctx: TransformerContext): void => {
         // Create the object type.
-        ctx.addObject(def)
+        // ctx.addObject(def)
+
+        let nonModelArray: ObjectTypeDefinitionNode[] = getNonModelFields(
```

Comment:
If this method returns ObjectTypeDefinitionNodes, I would change the name of the method to make it more clear.